src/iso19111/operation/singleoperation.cpp

```diff
@@ -2883,6 +2883,36 @@ bool SingleOperation::exportToPROJStringGeneric(
         return true;
     }
 
+    if (methodEPSGCode == EPSG_CODE_METHOD_SIMILARITY_TRANSFORMATION) {
+        const double XT0 =
+            parameterValueMeasure(
+                EPSG_CODE_PARAMETER_ORDINATE_1_EVAL_POINT_TARGET_CRS)
+                .value();
+        const double YT0 =
+            parameterValueMeasure(
+                EPSG_CODE_PARAMETER_ORDINATE_2_EVAL_POINT_TARGET_CRS)
+                .value();
+        const double M =
+            parameterValueMeasure(
+                EPSG_CODE_PARAMETER_SCALE_FACTOR_FOR_SOURCE_CRS_AXES)
+                .value();
+        const double q = parameterValueNumeric(
+            EPSG_CODE_PARAMETER_ROTATION_ANGLE_OF_SOURCE_CRS_AXES,
+            common::UnitOfMeasure::RADIAN);
+
+        // Do not mess with axis unit and order for that transformation
```

Comment:
Does it mean that if I use it in a DERIVEDPROJCRS in the DERIVINGCONVERSION, the axis unit and order of the CS are ignored?

Reply:
> Does it mean that if I use it in a DERIVEDPROJCRS in the DERIVINGCONVERSION, the axis unit and order of the CS are ignored?

yes I wasn't too sure about what to do here, so I did the same as for Affine